python/plugins/db_manager/dlg_sql_window.py

```diff
@@ -381,6 +381,7 @@ def updateUiWhileSqlExecution(self, status):
 
     def executeSqlCanceled(self):
         self.btnCancel.setEnabled(False)
+        self.btnCancel.setText(self.tr("Canceling, please wait ..."))
```

Comment:
What about just "Canceling" and leave the "please wait" bit for a tooltip? Otherwise the button will grow very large for this lengthy string

Reply:
I won't put up an argument but here are two reasons and a half for leaving as it is:

0.5: after https://github.com/OSGeo/gdal/pull/2953 the cancellation will happen so fast for GPKG/SL that most of the times nobody will notice the button change, for PG the cancellation will already be immediate after this PR
1. there is plenty of horizontal space in that part of the dialog, at the left of the button there is a huge progress bar
2. by changing the size and the phrasing of the button drastically we reduce the likelihood that the user misses the change

but, as I said it's not so important to me.

Reply:
Let's let @nirvn make the call?

Reply:
+1 to "Canceling" without the please wait suffix.

Reply:
+1 and I think we should keep the ellipsis … as an indicator that something is going on (unless we can add an animated spinner).

Reply:
For the spinner, I suspect the cursor is already set to spinner when we click on [ Execute SQL ]. If it's not, +1 to spinner too.

Reply:
Ideally, the cursor shouldn't spin after a click onto "Execute SQL" if it's still possible to click on "Cancel". If any, I would suggest to add spinners and ellipseses directly onto GUI elements to put them into context.

Reply:
True that.